opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/dto/KeyDTO.java

```diff
@@ -0,0 +1,9 @@
+package de.adorsys.opba.protocol.facade.dto;
+
+import lombok.Value;
+
+@Value
+public class KeyDTO {
```

Comment:
Please use KeyDto
no sense to highlight DTO that much

Reply:
ok